graphql/database/users/logTab.js

```diff
@@ -75,28 +75,28 @@ const logTab = async (userContext, userId, tabId = null, isV4 = true) => {
   } catch (e) {
     throw e
   }
+
   const todayTabCount = getTodayTabCount(user) + 1
   const isValid = isTabValid(todayTabCount, user.lastTabTimestamp)
 
-  // Update the user's counter for max tabs in a day.
-  // If this is the user's first tab today, reset the counter
-  // for the user's "current day" tab count.
-  // If today is also the day of all time max tabs,
-  // update the max tabs day value.
-  const isTodayMax = todayTabCount >= user.maxTabsDay.maxDay.numTabs
-  const maxTabsDayVal = {
-    maxDay: {
-      date: isTodayMax
-        ? moment.utc().toISOString()
-        : user.maxTabsDay.maxDay.date,
-      numTabs: isTodayMax ? todayTabCount : user.maxTabsDay.maxDay.numTabs,
-    },
-    recentDay: {
-      date: moment.utc().toISOString(),
-      numTabs: todayTabCount,
-    },
+  if (user.has('currentMissionId') && isValid) {
+    mission = await MissionModel.get(userContext, currentMissionId)
+    if (!mission.has('completed')) {
+      userMission = await UserMissionModel.query(userContext, currentMissionId, userId)
```

Comment:
@jedtan lmk if you have questions setting up dummy data in your local dynamodb

Reply:
also you'll need to user an override on these.  I think you can just user UserMissionModel.get(override,missionId,userId) here.  If using query, you'll need to call usingIndex(index)..execute() in order for the query to return anything